api/source/service/mysql/STIGService.js

```diff
@@ -293,26 +293,21 @@ Generalized queries for a single Rule, optionally with Check and Fix
 exports.queryRules = async function ( ruleId, inProjection ) {
   let columns = [
     'r.ruleId',
-    'r.title',
     'r.version',
+    'r.title',
     'r.severity',
-    'r.weight',
-    'r.vulnDiscussion',
-    'r.falsePositives',
-    'r.falseNegatives',
-    'r.documentable',
-    'r.mitigations',
-    'r.severityOverrideGuidance',
-    'r.potentialImpacts',
-    'r.thirdPartyTools',
-    'r.mitigationControl',
-    'r.responsibility'
+    'g.groupId',
+    'g.Title as "groupTitle"'
   ]
 
   let joins = [
-    'rule r'
-  ]
-
+    'revision rev ',
```

Comment:
Please rework these joins as discussed... no need for revision table

Reply:
reworked order of the tables in the joins